### PR TITLE
Initial work toward converting the FlutterView to use a FlutterImageView on demand

### PR DESCRIPTION
**Pull request by jason-simmons**

### CODE REVIEW

File: shell/platform/android/io/flutter/embedding/android/FlutterImageView.java [hunk 1]
```diff
@@ -56,51 +91,40 @@ public void acquireLatestImage() {
...
+      if (imagePlanes.length != 1) {
+        return;
+      }
+
+      final Plane imagePlane = imagePlanes[0];
+      final int desiredWidth = imagePlane.getRowStride() / imagePlane.getPixelStride();
+      final int desiredHeight = currentImage.getHeight();
+
+      currentBitmap =
+          android.graphics.Bitmap.createBitmap(
```
Comment (blasten) [inline shell/platform/android/io/flutter/embedding/android/FlutterImageView.java:124]: nit: `Bitmap.createBitmap`

Feel free to address this later: The Android team suggested to reuse the bitmap.

File: shell/platform/android/io/flutter/embedding/android/FlutterImageView.java [hunk 2]
```diff
@@ -46,6 +50,37 @@ public FlutterImageView(@NonNull Context context, @NonNull ImageReader imageRead
...
+    if (this.flutterRenderer != null) {
+      this.flutterRenderer.stopRenderingToSurface();
+    }
+
+    this.flutterRenderer = flutterRenderer;
+    flutterRenderer.startRenderingToSurface(imageReader.getSurface());
+  }
+
+  /**
+   * Invoked by the owner of this {@code FlutterTextureView} when it no longer wants to render a
```
Comment (blasten) [inline shell/platform/android/io/flutter/embedding/android/FlutterImageView.java:74]: `FlutterImageView`

File: shell/platform/android/io/flutter/embedding/android/FlutterImageView.java (hunk 1 above)
Comment (jason-simmons) [reply shell/platform/android/io/flutter/embedding/android/FlutterImageView.java]: done

File: shell/platform/android/io/flutter/embedding/android/FlutterImageView.java (hunk 2 above)
Comment (jason-simmons) [reply shell/platform/android/io/flutter/embedding/android/FlutterImageView.java]: done

**Pull request merged**